pkg/prompts/prompts.go

```diff
@@ -31,66 +27,27 @@ func RunPromptsFromConfigWithSkips(config *config.DraftConfig, varsToSkip []stri
 	inputs := make(map[string]string)
 
 	for _, customPrompt := range config.Variables {
+		promptVariableName := customPrompt.Name
 		if _, ok := skipMap[customPrompt.Name]; ok {
```

Comment:
should use promptVariableName as the key here.

Reply:
good catch thanks